contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -3,6 +3,8 @@ var _ = require("underscore");
 var BaseViews = require("edit_channel/views");
```

Comment:
Mainly just added handling for determining if the current selection is an exercise, setting the mastery criteria, and detecting changes on the assessment items